src/structs/Webhook.ts

```diff
@@ -47,7 +46,7 @@ export class Webhook {
   }
 
   private _formatIncoming (body): WebhookPayload {
-    if (body?.query?.length > 0) body.query = qs.parse(body.query.substr(1))
+    if (body?.query?.length > 0) body.query = Object.fromEntries(new URLSearchParams(body.query.substr(1)))
```

Comment:
```suggestion
    if (body?.query?.length > 0) body.query = Object.fromEntries(new URLSearchParams(body.query))
```
shouldn't need this anymore